docs/tutorial/language-tut.md

```diff
@@ -142,7 +142,7 @@ But now we're controlling things with an execution schedule.
 
 If you're curious to see how the Calyx compiler lowers this program to a Verilog-like structural form of Calyx, you can do this:
 
-    fud exec language-tutorial-mem.futil --to calyx-lowered
+    calyx language-tutorial-mem.futil
```

Comment:
This might be silly but just making sure, this only works if you installed the crate right?

Reply:
If you installed the executable, yes. I guess I hope that's not too much to assume…